docs/section19/Rule19-23.md

```diff
@@ -87,27 +92,31 @@
     - Case 5: Else if space type is dwelling unit and one or more of the design heating schedules fail per above then fail : `elif is_dwelling_unit == true and all(inf_pass_heating, occ_pass_heating, int_lgt_pass_heating, misc_pass_heating) == false: outcome = "FAIL" and raise_message "<Insert space.id> appears to be a dwelling unit and does not appear to have followed this rule per Section G3.1.2.2.1 for one more more of the following heating design schedules (cooling design schedules fall under the exception to Section G3.1.2.2.1 and were not assessed for dwelling units in this check): infiltration, occupants, lighting, gas and electricity using equipment."`  
-    - Case 6: Else if the space type is dwelling unit and all design heating schedules pass then the outcome is Pass: `elif is_dwelling_unit == true and all(inf_pass_heating, occ_pass_heating, int_lgt_pass_heating, misc_pass_heating) == true: outcome = "PASS"`  
-    - Case 7: Else if the space type is not defined and the building type is not multifamily and all schedules pass per above then pass: `elif is_dwelling_unit == false and is_space_type_defined == false and building_area_is_MF == false and all(inf_pass_heating, inf_pass_cooling,occ_pass_heating, occ_pass_cooling,int_lgt_pass_heating, int_lgt_pass_cooling,misc_pass_heating, misc_pass_cooling) == true: outcome = "PASS" and raise_message "Pass unless the space type is dwelling unit (the space type was not defined in the RMD schema and was assumed not to be a dwelling unit). Dwelling units fall under the exception to Section G3.1.2.2.1.`  
+    - Case 6: Else if the space type is dwelling unit and all design heating schedules pass then the outcome is UNDETERMINED: `elif is_dwelling_unit == true and all(inf_pass_heating, occ_pass_heating, int_lgt_pass_heating, misc_pass_heating) == true: outcome = "UNDETERMINED and raise_message "<Insert space.id> appears to be a dwelling unit and meets the requirements of this rule for heating design schdules. Cooling design schedules fall under the exception to Section G3.1.2.2.1 and were not assessed for this check. Conduct a manual review of cooling design schedules for infiltration, occupants, lighting, gas and electricity using equipment."`  
+    - Case 7: Else if the space type is not defined and the building type is not multifamily and all schedules pass per above then UNDETERMINED: `elif is_dwelling_unit == false and is_space_type_defined == false and building_area_is_MF == false and all(inf_pass_heating, inf_pass_cooling,occ_pass_heating, occ_pass_cooling,int_lgt_pass_heating, int_lgt_pass_cooling,misc_pass_heating, misc_pass_cooling) == true: outcome = "UNDETERMINED" and raise_message "Pass unless the space type is dwelling unit. Dwelling units fall under the exception to Section G3.1.2.2.1.`  
```

Comment:
Do we also need to update the Case 7 message?

Reply:
I think it is okay because the outcome itself is UNDETERMINED. The message is just saying that it passes unless it is a dwelling unit. Please let me know if you disagree.

Reply:
@yunjoonjung-PNNL The issue is that is_space_defined and building_area_is_MF are also equal False so there is no definitive way for us to actually know whether it is a dwelling unit hence the outcome as is. All we know is that is_dwelling_unit was never changed from False to True but we do not know definitively whether it is a dwelling unit or not.

Reply:
@claperle I cannot find G3.1.2.2.1

Reply:
@supriyagoel See below.
![image](https://github.com/user-attachments/assets/498e612a-e79b-4c7b-8367-c9b1687d8d24)